.env.example

```diff
@@ -0,0 +1,10 @@
+export USGS_USERNAME=something_secret_goes_here
+export USGS_PASSWORD=something_secret_goes_here
+
+export PANOPTES_PROD_CLIENT_ID=something_secret_goes_here
```

Comment:
I agree, though I think that that is outside the scope of this change set. I'll put it on the list for the next change set!

Reply:
Hm; well, this is sort of an interim step.

When I inherited theia, she could only connect to production version of panoptes. Not staging. I would like to get to the point where she can also upload subjects to staging, and I think we're allmost there. 

In the meantime, I have this set up so that I can switch between pointing at projects on prod Panoptes and staging Panoptes with my local theia, and not have to keep copy and pasting over the keys as I am developing.

For deployment, we _can_ collapse them.